src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -420,21 +420,30 @@ private EmailWrapper generateFeedbackSessionEmailBaseForStudents(
         return email;
     }
 
+    private String generateInstructorFragment(String courseId, String courseName) {
+
+        String fragmentParameters = SanitizationHelper.sanitizeForHtml(courseId) + ", "
+                + SanitizationHelper.sanitizeForHtml(courseName);
```

Comment:
- `fragmentParameters` is too generic and does not suggest what the variable is used for. Judging from the usage, `courseIdentifier` could be a better name.
- Since the course identifier is presented as `[courseId] courseName` in other parts of the email, I suggest the same formatting to be applied here.